src/api/customer/customer-serializer.ts

```diff
@@ -19,6 +19,57 @@ const createCustomerSerializer: CustomerCreationRequestHandler = (req, res, next
     next();
 }
 
+const findCustomerSerializer: CustomerFindOneRequestHandler = (req, res, next) => {
+    const { customerFind } = res.locals;
+    if (customerFind !== null) {
```

Comment:
Sabemos que a constante `customerFind` vai ser um objeto caso haja valor.
Poderíamos simplificar e alterar a comparação apenas para saber se é um `falsy value`.

```suggestion
    if (customerFind) {
```